packages/neos-ui/src/Containers/RightSideBar/Inspector/index.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import {produce} from 'immer';
 import mapValues from 'lodash.mapvalues';
 import {connect} from 'react-redux';
-import {$get, $contains, $set} from 'plow-js';
```

Comment:
biggest merge conflict was in this file (packages/neos-ui/src/Containers/RightSideBar/Inspector/index.js)

because of @JamesAlias and @markusguenther work

https://github.com/neos/neos-ui/pull/3304
https://github.com/neos/neos-ui/pull/2989

Reply:
As you @grebaldi also did some refactoring here (extract preprocessViewConfiguration into evaluateExpression), i suspect we need to migrate this file again and start fresh